pkg/arvo/sur/group.hoon

```diff
@@ -0,0 +1,94 @@
+^?
+|%
+::  $group-id: unique identifier for a group
+::
++$  group-id  [=ship =term]
+::  $groups: a mapping from group-ids to groups
+::
++$  groups  (map group-id group)
+::  $group-tag: an identifier used by groups
+::
+::    These tags should have precise semantics, as they are shared across all
+::    apps.
+::
++$  group-tag  ?(role-tag)
```

Comment:
Looks like `group-tag` can just go for now, it doesn't really add anything on top of `role-tag`. Naming of the latter is more precise, too.

Reply:
what if we want to add more tags later?

Reply:
We move to `+$  group-tag  ?(role-tag new-tag)`, and the existing uses of `role-tag`s will still nest fine. The current `group-tag` is exactly identical to `role-tag` and so superfluous, is what I'm saying.